3-extra/2-sorting-algorithm.js

```diff
@@ -14,7 +14,17 @@ You don't have to worry about making this algorithm work fast! The idea is to ge
 "think" like a computer and practice your knowledge of basic JavaScript.
 */
 
-function sortAges(arr) {}
+
+function sortAges(arr) {
+  return arr
+    .filter((element) => {
```

Comment:
Tests pass on this file but instructions do say "removes any non-number data types without using the built-in javascript filter method"